samples/subsys/ipc/ipc_service/static_vrings/remote/prj.conf

```diff
@@ -2,7 +2,6 @@ CONFIG_PRINTK=y
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 
 CONFIG_IPC_SERVICE=y
-CONFIG_IPC_SERVICE_BACKEND_RPMSG=y
```

Comment:
this could be split into another commit, since you are only changing the default in kconfig so keeping it here won't break any build, right?